ngsderive/commands/endedness.py

```diff
@@ -53,22 +52,19 @@ def resolve_endedness(
     if neither > 0 and both > 0:
         result["Endedness"] = "Unknown"
         return result
-    else:
-        assert neither == 0 and both == 0
+    assert neither == 0 and both == 0
```

Comment:
I'm generally not a big fan of asserting in production code, but if you're going to do it, you should include the reason for the assertion. E.g.

```suggestion
    assert neither == 0 and both == 0, "Reason"
```